rise/static/main.js

```diff
@@ -519,21 +522,48 @@ define([
                     async: true,
-                    condition: function() { return !!document.body.classList; } }
-                ]
+                    condition: function() { return !!document.body.classList; },
+                  },
+                ],
+
               };
+
               for (let setting of inherited) {
                 options[setting] = complete_config[setting];
               }
 
-              // Set up the Leap Motion integration if configured
-              var leap = complete_config.enable_leap_motion;
-              if (leap !== undefined) {
+              ////////// set up the leap motion integration if configured
+              var enable_leap_motion = complete_config.enable_leap_motion;
+              if (enable_leap_motion) {
                 options.dependencies.push({ src: require.toUrl('./reveal.js/plugin/leap/leap.js'), async: true });
-                options.leap = leap;
+                options.leap = enable_leap_motion;
               }
 
-              Reveal.configure(options);
+              ////////// set up chalkboard if configured
+              let enable_chalkboard = complete_config.enable_chalkboard;
+              if (enable_chalkboard) {
+                options.dependencies.push({ src: require.toUrl('./reveal.js/plugin/chalkboard/chalkboard.js'), async: true });
+                // xxx need to explore the option of registering jupyter actions 
+                // and have jupyter handle the keyboard entirely instead of this approach
+                // could hopefully avoid conflicting behaviours in case of overlaps
+                $.extend(options.keyboard,
+                         {
+                           // for chalkboard; also bind uppercases just in case
+                           63:  KeysMessager,                                        // '?' show our help
+                           61:  function() { RevealChalkboard.reset() },             // '=' reset chalkboard data on current slide
+                           45:  function() { RevealChalkboard.clear() },             // '-' clear full size chalkboard
+                           67:  function() { RevealChalkboard.toggleChalkboard() },  // 'C' toggle full size chalkboard
+                           99:  function() { RevealChalkboard.toggleChalkboard() },  // 'c' toggle full size chalkboard
+                           78:  function() { RevealChalkboard.toggleNotesCanvas() }, // 'n' toggle notes (slide-local) 
+                           110: function() { RevealChalkboard.toggleNotesCanvas() }, // 'N' toggle notes (slide-local)
+                           68:  function() { RevealChalkboard.download() },          // 'D' download recorded chalkboard drawing
+                           100: function() { RevealChalkboard.download() },          // 'd' download recorded chalkboard drawing
+                           // when 'd' is pressed
+                         });
+              }
+
+
+              Reveal.initialize(options);
```

Comment:
Can you remind me why you are using `initialize`here instead of `configure`?

Reply:
This is a point that the original author had pointed out, so in his own words:

> `Reveal.configure` does not load dependencies